fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -488,6 +488,18 @@ public class FunctionSet {
             .add(FunctionSet.FIRST_VALUE_REWRITE)
             .build();
 
+    public static final Set<String> varianceFunctions = ImmutableSet.<String>builder()
```

Comment:
```suggestion
    public static final Set<String> VARIANCE_FUNCTIONS = ImmutableSet.<String>builder()
```

Reply:
@Seaven 
other function name sets also use  lower camel case, for examples:
```
   public static final Set<String> decimalRoundFunctions =
            ImmutableSet.<String>builder()
                    .add(TRUNCATE)
                    .add(ROUND)
                    .add(DROUND)
                    .build();

    public static final Set<String> nonDeterministicFunctions =
            ImmutableSet.<String>builder()
                    .add(RAND)
                    .add(RANDOM)
                    .add(UUID)
                    .add(SLEEP)
                    .build();

    public static final Set<String> onlyAnalyticUsedFunctions = ImmutableSet.<String>builder()
            .add(FunctionSet.DENSE_RANK)
            .add(FunctionSet.RANK)
            .add(FunctionSet.NTILE)
            .add(FunctionSet.ROW_NUMBER)
            .add(FunctionSet.FIRST_VALUE)
            .add(FunctionSet.LAST_VALUE)
            .add(FunctionSet.FIRST_VALUE_REWRITE)
            .build();

```